src/content/docs/features/window-customization.mdx

```diff
@@ -2,13 +2,207 @@
 title: Window Customization
 ---
 
-import Stub from '@components/Stub.astro';
+Tauri provides lots of options for customizing the look and feel of your app's window. You can create custom titlebars, have transparent windows, enforce size constraints, and more.
 
-<Stub>
+## Configuration
 
-    - Based on https://tauri.app/v1/guides/features/window-customization
-    - Take
-    into account feedback in
-    https://github.com/tauri-apps/tauri-docs/issues/1344#issuecomment-1701912344
+There are three ways to change the window configuration:
 
-</Stub>
+- [Through tauri.conf.json](/references/v2/config/#windowconfig)
+- [Through the JS API](/references/v2/js/core/namespacewindow/#window)
+- [Through the Window in Rust](https://docs.rs/tauri/2.0.0-beta/tauri/window/struct.Window.html)
+
+## Usage
+
+- [Creating a Custom Titlebar](#creating-a-custom-titlebar)
+- [(macOS) Transparent Titlebar with Custom Window Background Color](#macos-transparent-titlebar-with-custom-window-background-color)
+
+### Creating a Custom Titlebar
+
+A common use of these window features is creating a custom titlebar. This short tutorial will guide you through that process.
+
+#### tauri.conf.json
+
+To remove the default titlebar, you'll need to set `decorations` to `false` in your `tauri.conf.json`:
+
+```json title="tauri.conf.json" {4}
+"tauri": {
+	"windows": [
+		{
+			"decorations": false
+		}
+	]
+}
+```
+
+#### Permissions
+
+To make calls to `appWindow` work, don't forget to add window permissions in capability file:
```

Comment:
```suggestion
Add window permissions in capability file:
```

Along the lines of other guides
`By default all plugin commands are blocked and cannot be accessed. You must define a list of permissions in your capabilities configuration.`